packages/react-components/src/components/__tests__/events-table-component-tests.tsx

```diff
@@ -0,0 +1,243 @@
+import * as React from 'react';
+import { cleanup, render, fireEvent, screen } from '@testing-library/react';
+import { create } from 'react-test-renderer';
+import { mount } from 'enzyme';
+import { TableOutputComponent } from '../table-output-component';
+import { CellRenderer, LoadingRenderer, SearchFilterRenderer } from '../table-renderer-components';
+import { AbstractOutputProps } from '../abstract-output-component';
+import { TimeGraphUnitController } from 'timeline-chart/lib/time-graph-unit-controller';
+import { TimeRange } from 'traceviewer-base/lib/utils/time-range';
+import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
+import { AgGridReact } from 'ag-grid-react';
+import { ColDef, Column, ColumnApi, GridApi, IRowModel, RowNode } from 'ag-grid-community';
+
+afterEach(() => {
+    cleanup
+    jest.clearAllMocks();
+});
+
+describe('Table output component tests', () => {
+    
+    test('Renders AG-Grid table with provided props & state', async () => {
+        const tableOutputComponentProps: AbstractOutputProps = {
+            tooltipComponent: null,
+            style: {
+                width: 0,
+                chartWidth: 0,
```

Comment:
Sorry for taking so long to review. The AbstractOutputProps have been updated and some of these properties have changed. The unit test fails to compile on latest master.

Reply:
@PatrickTasse Rebased on top of master. The unit tests should pass now